### ISSUE TITLE
Obstructions .get_by_id not working with Integers anymore

### ISSUE THREAD

**Issue by C-Rogge** [FireDynamics/fdsreader#58]
Today is just a quick one:

When getting an obstructions by the _.get_by_id()_-Command, only strings seem to work now.
For example calling _.get_by_id('1')_ yields the right result, while .get_by_id(1) returns _None_.
Im not sure if this is intended, but at least some versions ago also integers worked.

This should be reproduceable in pretty much any simulations with obstructions right now, but before @lu-kas keeps scolding me: [fdsreaderTestcase .fds.txt](https://github.com/FireDynamics/fdsreader/files/10426040/fdsreaderTestcase.fds.txt)

**Comment by JanVogelsang** [comment]
Hm, actually I am not sure if I want to "fix" this. IDs are supposed to be strings as they are not limited to numbers, but can be any alphanumeric character (or whatever the ID-limitations of FDS are).  
@lu-kas What do you think?
I could of course just check if obstruction ID contains only numeric characters and then compare it to the integer parameter, but at the same time there is nothing stopping you from just going with `.get_by_id(str(1))`.  

If it makes your life easier I can implement that, though, doesn't really hurt.

**Comment by C-Rogge** [comment]
To be honest it doesn't really matter to me. Like you said, its easy to just convert the integer into a string. 
I just stumbled over the changed behavior as some of my code did not work anymore since i last updated the fds reader, so i thought i as well just ask if the change was intended or not.

I hope for the next fds/smv version to actually name my obstructions for better recognition while analysing with the reader, as we discussed in another thread, so no need to (re-)implement this for me. Thanks for looking into it anyway ;)
